src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 
 ############# Database Properties ###########################################
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.url=jdbc:postgresql://postgres:5432/postgres
```

Comment:
must change the localhost to postgres to reflect the host network in the docker container